network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/LineTest.java

```diff
@@ -23,4 +33,41 @@ public void testTieLineAdder() {
     public void baseAcLineTests() {
         // FIXME TO FIX LATER
     }
+
+    @Test
+    public void isOverloadedTest() {
```

Comment:
@etiennehomer there is no already existing and passing test in the TCK for overloads?